### PR TITLE
WMS-587 | Fix inconsistencies between Figma's mobile spaces and current implementation

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?
Fix inconsistencies between Figma's mobile spaces and current implementation:
* Defines the button's style in the mobile's header;
* Allow customizing the mobile's stacked header's title;
* Update mobile's header's sandwich size and color;
* Center mobile's stacked header title;
* Force mobile's stacked header elliptization;
* Update SelectList's action bar padding.

#### :man_shrugging: Why?
Cause it differed from Figma.

#### :pushpin: Jira Issue
[WMS-587](https://paacklogistics.atlassian.net/browse/WMS-587)

#### :no_good: Blocked by
**Comments in Figma**

#### :clipboard: Pending
Nothing

### :fire: Extra
This JIRA card was smaller, but those changes are all in the same Figma screen/component.

### CODE REVIEW

File: src/UI/Internal/SideBar.elm [hunk 1]
```diff
@@ -110,26 +112,45 @@ viewHead cfg (Menu.Menu prop _) title maybeStack =
...
-                , Element.inFront mobileHeadSandwich
-                ]
-                [ Element.el
-                    [ Element.width fill, Font.center, padding 20 ]
-                    (Text.heading5 title |> Text.renderElement cfg)
-                ]
+            Element.el
+                (baseAttrs
+                    |> (::) (Element.inFront mobileHeadSandwich)
+                )
```
Comment (renanpvaz) [inline src/UI/Internal/SideBar.elm:132]: why not just? 
```elm
Element.inFront mobileHeadSandwich :: baseAttrs

**Review by renanpvaz — APPROVED** [review]

**Review by miguelcoba — APPROVED** [review]

**Comment by PedroHLC** [comment]
I've applied Sufiyan's fixes and refactored that part out-of-the SideBar

File: src/UI/Internal/Button.elm [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                                ActionRedirect link ->
+                                    Link.wrapElement renderConfig attributes link
+
+                        _ ->
+                            identity
+                   )
+
+        Toggle _ _ ->
+            -- TODO: Do it if you need it someday...
+            Element.none
```
Comment (renanpvaz) [inline src/UI/Internal/Button.elm:130]: can't we make this any flatter? merging some of the cases or using `let-in`

**Review by renanpvaz — APPROVED** [review]

File: src/UI/Layout/SplitSelectable.elm [hunk 3]
```diff
@@ -21,8 +21,8 @@ type alias Config object msg =
 
 
 type alias MobileConfig msg =
-    { buttons : List (Button msg)
-    , title : String
+    { rightButton : Maybe (Button msg)
```
Comment (renanpvaz) [inline src/UI/Layout/SplitSelectable.elm:24]: why was this change necessary? Are we not supplying more than one button in some cases?

Comment (PedroHLC) [reply src/UI/Layout/SplitSelectable.elm:24]: I didn't found any case. And when there are no buttons, Sufyan centers the title as if there was one invisible button in the right (or ignores the button on left for alignment).
